emoji-selector@maestroschan.fr/emojiCategory.js

```diff
@@ -158,7 +164,7 @@ var EmojiCategory = class EmojiCategory {
 
 	_openCategory() {
 		Extension.GLOBAL_BUTTON.clearCategories();
-		this.super_item.label.text = this.categoryName;
+		this.super_item.label.text = _( this.categoryName  );
```

Comment:
the category names are in the .pot file because they're already marked as translated in other parts of the code (probably at the beginning of extension.js), i didn't notice it was broken but it should be fixed properly in extensions.js instead of importing gettext multiple times to re-translate strings which should be already translated at this point of the execution.

Reply:
if you're not sure how to fix it, just ignore my explanation and i'll fix the issue myself after merging your translation